src/simularium/VisGeometry/index.ts

```diff
@@ -29,32 +29,35 @@ import jsLogger from "js-logger";
 import { ILogger, ILogLevel } from "js-logger";
 import { cloneDeep, noop } from "lodash";
 
-import VisAgent from "./VisAgent";
-import VisTypes from "./VisTypes";
-import PDBModel from "./PDBModel";
-import FrontEndError from "./FrontEndError";
+import VisAgent from "../VisAgent";
+import VisTypes from "../VisTypes";
+import PDBModel from "../PDBModel";
+import FrontEndError from "../FrontEndError";
 
-import { DEFAULT_CAMERA_Z_POSITION, DEFAULT_CAMERA_SPEC } from "../constants";
+import {
+    DEFAULT_CAMERA_Z_POSITION,
+    DEFAULT_CAMERA_SPEC,
+} from "../../constants";
 import {
     TrajectoryFileInfo,
     CameraSpec,
     EncodedTypeMapping,
     AgentDisplayDataWithGeometry,
-} from "./types";
-import { AgentData } from "./VisData";
-
-import SimulariumRenderer from "./rendering/SimulariumRenderer";
-import { InstancedFiberGroup } from "./rendering/InstancedFiber";
-import { InstancedMesh } from "./rendering/InstancedMesh";
-import { LegacyRenderer } from "./rendering/LegacyRenderer";
-import GeometryStore, { DEFAULT_MESH_NAME } from "./VisGeometry/GeometryStore";
+} from "../types";
+import { AgentData } from "../VisData";
+
+import SimulariumRenderer from "../rendering/SimulariumRenderer";
+import { InstancedFiberGroup } from "../rendering/InstancedFiber";
+import { InstancedMesh } from "../rendering/InstancedMesh";
+import { LegacyRenderer } from "../rendering/LegacyRenderer";
```

Comment:
VisGeometry is the only one that imports all these `rendering/` files. It looks odd that they became further removed from VisGeometry.  Do you have any objection to my doing a followup that brings all the visualization code together and starts to separate it from the controller/simulation code?  Or doing it in this PR.

Reply:
you mean just moving the rendering folder into VisGeometry/ too? Or more extensive changes. The former seems fine for this PR, the latter maybe separate PR

Reply:
Yes - please just move rendering/ into VisGeometry and we can do the rest in a separate PR! thank you

Reply:
The `worker/KMeansWorker` imports `rendering/KMeans3d` just FYI